generators/generator-bot-enterprise-people/generators/app/templates/README.md

```diff
@@ -1,6 +1,6 @@
 # Welcome to your new bot
 
-This Bot Project was created using the **[Enterprise People Bot](https://aka.ms/EnterprisePeopleBot)** template. You **must complete the configuration steps outlined below for your bot to function.** 
+This Bot Project was created using the [Enterprise People Bot](https://aka.ms/EnterprisePeopleBot) template. You **must complete the configuration steps outlined below for your bot to function.**
 
 ## Configure your Enterprise Assistant
```

Comment:
Should this be called "Configured your Enterprise People Bot" instead of "Enterprise Assistant"? That's because Enterprise Assistant is the one that is the consumer of this skill.